octobot_trading/data/order.py

```diff
@@ -175,6 +175,12 @@ async def initialize_impl(self):
         Initialize order status update tasks
         """
         if not self.exchange_manager.is_backtesting:
+            if not self.exchange_manager.is_simulated:
+                # Create a task to synchronize order data with exchange
+                asyncio.create_task(get_chan(ORDERS_CHANNEL, self.exchange_manager.id).get_internal_producer().
+                                    update_order_from_exchange(self))
```

Comment:
Not awaiting this task will make it so that the following call (`asyncio.get_event_loop().call_later(self.CHECK_ORDER_STATUS_AFTER_INIT_DELAY...`) will be executed after a delay that is independent from whether this `update_order_from_exchange` task is completed or not, is it what we want in real trading here?

Reply:
There is no link between those two methods, I think it's a correct behavior.